xrpl/models/transactions/transaction.py

```diff
@@ -468,3 +472,42 @@ def from_blob(tx_blob: str) -> Transaction:
             The formatted transaction.
         """
         return Transaction.from_xrpl(decode(tx_blob))
+
+    @classmethod
+    def from_xrpl(cls: Type[T], inp_params: Union[str, Dict[str, Any]]) -> T:
```

Comment:
```suggestion
    def from_xrpl(cls: Type[T], value: Union[str, Dict[str, Any]]) -> T:
```
This should use the same formatting as the function it's overriding.